doc/Examples.rst

```diff
@@ -529,21 +529,23 @@ no suffix.  The prefix itself is divided into two parts ("and" and
 
 
 
-Entity Mentions
+Entities
 ---------------
 
-Entity mentions are strings in the text that refer to named entities.
-A mention is identified by its character start/end offsets.  It has an
-entity type (e.g. PERSON, LOCATION).  It may have a normalized form
-(e.g. normalized whitespace, affixes removed), a confidence, and an
-in-document coreference chain id.  The chain id is the index of the
-head mention of the chain.  The head mention is the longest mention in
-the chain.  If multiple mentions are longest, the first one is chosen.
+Entities refer to "real world" entities. Each entity in a document is 
```

Comment:
I'm not comfortable with "Real World" here... This was the language originally used for Resolved Entities, which indeed had a real-world QID. Perhaps something like:

Entities are a result of document-level grouping of mentions that refer to the same thing.  
In the example below, the mentions ["George Bush", "Bush"] form a single entity (of type "PERSON").